tests/sdks/python/application/test_envilder_facade.py

```
@@ -150,7 +150,7 @@ def Should_RaiseValueError_When_FilePathIsEmpty(
         self,
     ) -> None:
         # Act
-        action = lambda: Envilder.from_file("")
+        action = lambda: Envilder.from_map_file("")
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `action` as a `def`

(E731)